Zend/zend_language_scanner.l

```diff
@@ -2032,6 +2032,10 @@ string:
 
 <INITIAL>"<?" {
 	if (CG(short_tags)) {
+		if (CG(short_tags_notice) == 0) {
+			zend_error(E_DEPRECATED, "PHP Short tags are deprecated as of PHP 7.4 and will be removed in a future version");
```

Comment:
We usually don't mention versions in deprecation messages.

Reply:
We may also mention that this message will only be printed once, even if there are more short tags:

> PHP short tags are deprecated. This message will not be printed for further short tags uses

Reply:
Is the new message which indicates the first usage alright?

Reply:
Looks good!